content/en/event-driven_architecture.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Event-Driven Architecture
+status: Feedback Appreciated
+category: concept
+---
+## What it is
+Event-driven architecture is a software architecture that promotes the creation, processing, and consumption of events. An event can be seen as any change of state or other significant occurrence in hardware or software. This architecture creates the structure in which events can be properly routed from their source to the desired receivers.
+## Problem it addresses
+As more data is becoming real time in nature, it becomes difficult to find reliable ways to ensure that events are captured then routed to the service that needs to process the event request. There is also the need to guarantee delivery of the event to the proper service.
```

Comment:
```suggestion
As more data becomes real-time, finding reliable ways to ensure that events are captured and routed to the service that must process the event request gets increasingly challenging. There is also the need to guarantee delivery of the event to the proper service.
```

Reply:
This seems rather vague: "There is also the need to guarantee delivery of the event to the proper service." Maybe something like "Some applications may even need an event delivery guarantee for specific services to function as designed." >> this may not be correct but I feel that "there is also a need" needs more context if that makes sense.

Reply:
I just pushed some code changes